jena-extras/jena-serviceenhancer/src/test/java/org/apache/jena/sparql/service/enhancer/impl/TestServiceEnhancerMisc.java

```diff
@@ -462,4 +471,55 @@ public void testNormalization01() {
         int actualRowCount = AbstractTestServiceEnhancerResultSetLimits.testWithCleanCaches(dataset, queryStr, 1000);
         Assert.assertEquals(4, actualRowCount);
     }
+
+    /**
+     * Test for <a href="https://github.com/apache/jena/issues/1688">JENA-1688</a>.
```

Comment:
JENA-1688 is a JIRA from 2019!
Try: "apache/jena/issues/1688"

Reply:
Right - updated as suggested :)